realm/realm-library/src/androidTestObjectServer/java/io/realm/SyncConfigurationTests.java

```diff
@@ -377,18 +378,17 @@ public void directory_dirIsAFile() throws IOException {
         file.delete(); // clean up
     }
 
-    /* FIXME: deleteRealmOnLogout is not supported by now
+    @Ignore("deleteRealmOnLogout is not supported yet")
     @Test
     public void deleteOnLogout() {
-        User user = createTestUser();
+        SyncUser user = createTestUser();
         String url = "realm://objectserver.realm.io/default";
 
         SyncConfiguration config = new SyncConfiguration.Builder(user, url)
-                .deleteRealmOnLogout()
+                //.deleteRealmOnLogout()
```

Comment:
I really don't know.  All of this code had been commented out.  Naturally, it no longer compiles.   Since the test is now @Ignored instead of being commented out, I had to add that one line comment because it doesn't compile and I don't want to delete information.  Grrrrrr.....